src/zero/module/copyfile.clj

```diff
@@ -18,12 +17,12 @@
   "Submit WORKFLOW to Cromwell in ENVIRONMENT."
   [environment workflow]
   (cromwell/submit-workflow
-   environment
-   (io/file (:top workflow-wdl))
-   nil
-   (-> workflow (select-keys [:src :dst]) (util/prefix-keys pipeline))
-   (util/make-options environment)
-   {}))
+    environment
+    (util/extract-resource (:top workflow-wdl))
```

Comment:
```suggestion
    (clojure.java.io/file (.getFile (clojure.java.io/resource (:top workflow-wdl))))
```
That'll create a file out of something on the classpath (so "wdl/copyfile.wdl" here) without copying it

Reply:
This test is a bit rubbish and isn't really testing the real runtime behaviour - when this file is bundled in the jar, your code won't work unfortunately (i tried it myself :) ).

Reply:
Fair enough. It is weird that the resources file isn't available (other things from resources load fine in the jar, like log4j2.xml in my logging branch) but I think what we have here works perfectly fine and we have little reason to spend more time tweaking it.

Reply:
Right, but I don't think you're reading from that file yourself? I guess log4j looks for that file on the classpath and handles it?